rule/context-as-argument.go

```diff
@@ -1,22 +1,49 @@
 package rule
 
 import (
+	"fmt"
 	"go/ast"
 
 	"github.com/mgechev/revive/lint"
 )
 
 // ContextAsArgumentRule lints given else constructs.
-type ContextAsArgumentRule struct{}
+type ContextAsArgumentRule struct {
+}
 
 // Apply applies the rule to given file.
-func (r *ContextAsArgumentRule) Apply(file *lint.File, _ lint.Arguments) []lint.Failure {
+func (r *ContextAsArgumentRule) Apply(file *lint.File, args lint.Arguments) []lint.Failure {
+	var allowTypesBefore []string
+	if len(args) > 1 {
```

Comment:
Can we have this panic if `len(args) != 1` so we can skip the second if statement?

Reply:
Sure, done